src/js/examples/es8-async-await.js

```diff
@@ -18,27 +18,36 @@ function napTime(duration) {
   });
 }
 
+
 /* 
-  Await requires a Promise to be returned and
-  can only be used inside an async function.
-*/ 
+  Await requires a Promise to be returned which we can await 
+  and store in const variable and make use of it, as below with try-catch-finally
+*/
 async function getFood() {
-  console.log('Blast off 🚀');
-
-  await fetchMeat()
-    .then((bacon) => {
-      console.log(bacon);
-    })
-    .catch((err) => {
-      console.error(err);
-    });
-
+  console.log("Blast off 🚀");
+
+  try {
+    const bacon = await fetchMeat();
```

Comment:
Please comment out this whole process to document why you might use this over `.then`

Reply:
added.